### PR TITLE
Popout pre-layout refactoring

### PR DESCRIPTION
**Pull request by overevkin**

### CODE REVIEW

File: js/core/form/editors/impl/dateTime/views/TimeView.js [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default Marionette.LayoutView.extend({
             }
         }
 
-        this.dropdownView = dropdown.factory.createDropdown({
+        this.popoutView = dropdown.factory.createPopout({
```
Comment (xSkara) [inline js/core/form/editors/impl/dateTime/views/TimeView.js]: What is the reason to replace it with popout?

Comment (overevkin) [inline js/core/form/editors/impl/dateTime/views/TimeView.js]: #197868, as long as it's intended to be used in CP

File: js/core/form/editors/impl/dateTime/views/TimeView.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ export default Marionette.LayoutView.extend({
                 })
             }),
             autoOpen: false,
-            panelPosition: 'down'
+            panelPosition: 'up'
```
Comment (xSkara) [inline js/core/form/editors/impl/dateTime/views/TimeView.js]: Seems like debug traces, should be removed.

Comment (overevkin) [inline js/core/form/editors/impl/dateTime/views/TimeView.js]: Surely

**Pull request merged**